packages/paste-website/src/pages/getting-started/engineering/index.mdx

```diff
@@ -336,6 +273,96 @@ This is a free font that is [hosted on Google Fonts](https://fonts.google.com/sp
 [how we load the font](https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/assets/css/fonts.css#L105)
 into your own app if you are using the default theme.
 
+<Box marginBottom="space140" />
+
+## Additional information
+
+### Core package list
+
+Core gives you the following packages. You can install these packages individually as well.
```

Comment:
> You can install these packages individually as well 

"as well" might read as "in addition to". You might want to caveat it slightly more.